stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -173,8 +173,9 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
   ///   that is layout compatible with `T`.
   /// - Postcondition: The memory is bound to 'T' starting at `self` continuing
   ///   through `self` + `count` * `MemoryLayout<T>.stride`
-  /// - Warning: Binding memory to a type is potentially undefined if the
-  ///   memory is ever accessed as an unrelated type.
+  /// - Warning: A memory location may only be bound to one type at a time.
+  ///   Accessing memory as type `U` while it is bound to an unrelated type `T`
+  ///   is undefined.
```

Comment:
You need to say, "The behavior of accessing..." or "type `T` invokes undefined behavior".  I like the former.